unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs

```diff
@@ -105,6 +105,21 @@ async UniTaskVoid LoadEmotesAsync(CancellationToken ct = default)
                     allEmotes.AddRange(embeddedEmotesSo.GetAllEmotes());
                     allEmotes.AddRange(await emotesCatalogService.RequestOwnedEmotesAsync(userProfileBridge.GetOwn().userId, ct) ?? Array.Empty<WearableItem>());
 
+                    Dictionary<string, WearableItem> consolidatedEmotes = new Dictionary<string, WearableItem>();
```

Comment:
if this method called often, then I would suggest to cache/Clear dictionary or use Unity pools 

```
using (DictionaryPool<string, WearableItem>.Get(out  Dictionary<string, WearableItem> consolidatedEmotes))
{
foreach (var emote in allEmotes) {...}
}
```

Reply:
Will do it on `dev`